### PR TITLE
change createPool validation for new contract #29

### PR DESCRIPTION
**Pull request by nadyasav**

### CODE REVIEW

File: frontend/src/components/createPool/CreatePool.tsx [hunk 1]
```diff
@@ -29,26 +55,32 @@ function CreatePool() {
...
-  const [anchorSelectState, setAnchorSelectState] = useState(false);
-  const [anchorSelected, setAnchorSelected] = useState<IToken | null>(null);
-
-  const [approveTokensSelect, setApproveTokensSelect] = useState(false);
-  const [approveTokensSelected, setApproveTokensSelected] = useState<Array<IToken>>([]);
-
-  const [inputValue, setInputValue] = useState(0);
+  const handleProfitPalsVaultCreatedEvent = (log: Log[]) => {
+    console.log('Created pool data: ', log);
+    navigate(`${ROUTES.assetManagement}/${(log as ILogs[])[0].args.vault}`);
```
Comment (utgarda) [inline frontend/src/components/createPool/CreatePool.tsx:64]: `log as ILogs[])[0].args.vault`
```typescript
const vaultCreateEvent = (log as ILogs[])[0]
const vaultAddress = vaultCreateEvent.args.vault
navigate(`${ROUTES.assetManagement}/${vault}`);
```

File: frontend/src/components/createPool/CreatePool.tsx [hunk 2]
```diff
@@ -29,26 +55,32 @@ function CreatePool() {
...
+  }, [status]);
+
+  useEffect(() => {
+    if (anchorSelected) {
+      const approvedTokens = getApprovedTokens();
+      if (approvedTokens.length >= 2 && approvedTokens.length <= tokensMaxCount && inputValue) {
+        setBtnSubmitDisabled(false);
+      } else {
+        setBtnSubmitDisabled(true);
+      }
```
Comment (utgarda) [inline frontend/src/components/createPool/CreatePool.tsx:81]:
```typescript
const isInputValid = approvedTokens.length >= 2 && approvedTokens.length <= tokensMaxCount && inputValue
setBtnSubmitDisabled(!isDataValid)
```

File: frontend/src/components/createPool/CreatePool.tsx [hunk 3]
```diff
@@ -92,14 +111,14 @@ function CreatePool() {
...
+  const handleSubmit = async () => {
     const approvedTokens = getApprovedTokens().map((item) => item.address);
     const dataParams = [
       anchorSelected?.address,
       approvedTokens,
       inputValue,
-      nativeNetworkToken.fullName,
-      nativeNetworkToken.name,
+      anchorSelected?.name,
+      anchorSelected?.name,
```
Comment (utgarda) [inline frontend/src/components/createPool/CreatePool.tsx:121]: (later) anchor+token1+token2...

File: frontend/src/components/createPool/CreatePool.tsx [hunk 4]
```diff
@@ -141,7 +160,7 @@ function CreatePool() {
               onClick={() => {
                 setApproveTokensSelect(true);
               }}
-              disabled={(!anchorSelected && true) || getApprovedTokens().length >= tokentMaxCount}
+              disabled={(!anchorSelected && true) || getApprovedTokens().length >= tokensMaxCount}
```
Comment (utgarda) [inline frontend/src/components/createPool/CreatePool.tsx:163]: .

**Review by utgarda — APPROVED** [review]
LGTM 
take care of comments and merge! :rocket:

**Review by pixelPepe — APPROVED** [review]

**Pull request merged**